source/features/useful-forks.tsx

```diff
@@ -17,13 +17,13 @@ function getUrl(): string {
 }
 
 async function init(): Promise<void | false> {
-	const forkCount = await elementReady('#repo-network-counter');
+	const forkCount = await elementReady('#Layout-main');
 	if (looseParseInt(forkCount) === 0) {
 		return false;
 	}
 
 	const selector = [
-		'#network', // `isRepoForksList`
+		'#Layout-main', // `isRepoForksList`
```

Comment:
Thanks for the PR! We should keep both selectors here so that it keeps working on the old view until it's finalized. 

The old selector can be commented with `// `isRepoForksList` before March 2023

Reply:
@fregante So do I change the `Layout-main` back to `isRepoForksList`?

Reply:
Both should stay. So just *add* the new one